Introductory Tests/traj_opt.ipynb

```diff
@@ -2,36 +2,39 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.        target_altitude = 5000**2           # Target altitude squared
</pre><p>would call target_squared_altitude to avoid mixups</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>